<ui version="4.0" >
 <class>helpForm</class>
 <widget class="QWidget" name="helpForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QTextBrowser" name="textBrowser" />
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="homeButton" >
       <property name="text" >
        <string>&amp;Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backButton" >
       <property name="text" >
        <string>&amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
